### PR TITLE
Pull request for unarchive-asterisk-tests

### PR DESCRIPTION
**Pull request by fblackburn1**
fix styling issues
surround file_description variable by double quote
unarchive post-wizard asterisk tests

### CODE REVIEW

File: wazo_acceptance/steps/asterisk.py [hunk 1]
```diff
@@ -14,9 +14,14 @@
...
 
     for row in context.table:
         row = row.as_dict()
         assert_that(hints, has_item(has_entries(row)))
+
+
+@then('Asterisk command "{command}" returns no error')
+def then_asterisk_command_group1_returns_no_error(context, command):
+    command = ['asterisk', '-rx', '"{command}"'.format(command=command)]
+    assert context.remote_sysutils.send_command(command)
```
Comment (fblackburn1) [inline wazo_acceptance/steps/asterisk.py]: I don't get this assert, if I send `patate core reload` the assert pass

Comment (sduthil) [reply wazo_acceptance/steps/asterisk.py]: I guess the point is to test if Asterisk is running correctly. If Asterisk is not running, the assert fails. But the assert is misleading, I'll make it clearer

File: features/daily/asterisk.feature [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Feature: Asterisk
 
     Scenario: Asterisk is correctly installed
-        Then Asterisk may open at most 8192 file descriptors
+        Then Asterisk may open at most "8192" file descriptors
```
Comment (sduthil) [inline features/daily/asterisk.feature]: The double quote is only here to allow spaces within the variable value. They are not strongly necessary. Do we have a convention to surround every variable with quotes? I don't have a strong opinion on this:
* no quotes is more natural to read _in this case_
* quotes give the hint "this is a variable"

What's your opinion on this?

In any case, we should write down this kind of convention in a DEVELOP.md file.

Comment (bloom1) [reply features/daily/asterisk.feature]: @sduthil we added a style guide in the README yesterday. @fblackburn1 added the variable quoting today.

Comment (sduthil) [reply features/daily/asterisk.feature]: Thanks :)

**Pull request merged**